test/test_testing.py

```diff
@@ -3337,6 +3337,32 @@ def test_get_pebble_methods(self):
         client = backend.get_pebble('/custom/socket/path')
         self.assertIsInstance(client, _TestingPebbleClient)
 
+    def test_reboot(self):
+        class RebootingCharm(ops.CharmBase):
+            def __init__(self, *args, **kwargs):  # type: ignore
+                super().__init__(*args, **kwargs)  # type: ignore
+                self.framework.observe(self.on.install, self._reboot)
+                self.framework.observe(self.on.remove, self._reboot_now)
+
+            def _reboot(self, event: ops.RemoveEvent):
```

Comment:
Looks like these types should be the other way around:

```suggestion
            def _reboot(self, event: ops.InstallEvent):
```

Reply:
Ah, thanks. Sloppy on my part (I had them around the other way originally and then changed my mind, but missed the type).